src/metabase/driver/mongo.clj

```diff
@@ -148,11 +150,12 @@
                       (describe-table-field field (field parsed-rows))))})))
 
 (defn- analyze-table [table new-field-ids]
+  ;; this is actually for caching values.
   ;; We only care about 1) table counts and 2) field values
-  {:row_count (analyze/table-row-count table)
+  {#_:row_count #_(cached-values/table-row-count table)
    :fields    (for [{:keys [id] :as field} (table/fields table)
-                    :when (analyze/test-for-cardinality? field (contains? new-field-ids (:id field)))]
-                (analyze/test:cardinality-and-extract-field-values field {:id id}))})
+                    #_:when #_(classify/test-for-cardinality? field (contains? new-field-ids (:id field)))]
```

Comment:
can we remove all the commented out stuff, or is there a reason to keep it around